v0/destinations/tiktok_ads/transform.js

```diff
@@ -99,8 +99,11 @@ const getTrackResponse = (message, Config, event) => {
 
   response.method = defaultPostRequestConfig.requestMethod;
   response.endpoint = TRACK_ENDPOINT;
-  response.body.JSON = removeUndefinedAndNullValues(payload);
-
+  // add partner name
+  response.body.JSON = removeUndefinedAndNullValues({
+    ...payload,
+    partner_name: "RudderStack"
```

Comment:
In spite of using `RudderStack`, we should create a constant variable in config.js and use that.